File: openapi-client/php-dt/src/App/DTO/SendMailAdvFrom.php
<?php
declare(strict_types=1);

namespace App\DTO;

use Articus\DataTransfer\PhpAttribute as DTA;

/**
 * The information to use for the From address in the email. from.
 */
class SendMailAdvFrom
{
    /**
     * The email address
     */
    #[DTA\Data(field: "email")]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $email = null;

    /**
     * (optional) Name to use for the sending contact.
     */
    #[DTA\Data(field: "name", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $name = null;

}

File: swagger-client/scala-akka-http-server/src/main/scala/io/swagger/server/model/SendMailAdv_from.scala
package io.swagger.server.model


/**
 * The information to use for the From address in the email. from.
 *
 * @param email The email address for example: ''business@company.com''
 * @param name (optional) Name to use for the sending contact. for example: ''The Man''
 */
case class SendMailAdv_from (
  email: String,
  name: Option[String]
)

